extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -111,6 +111,13 @@ export let SizeInfoDef;
     }} */
 let CreativeMetaDataDef;
 
+/** @typedef {{
+      rtcResponse: ?Object<string, *>,
+      rtcTime: number,
+      callout: string,
+      error: ?Error}} */
```

Comment:
I thought we were going to make this an enum?

Reply:
done